### DISCUSSION TITLE
Way to play on Home Assistant local device?

### DISCUSSION THREAD

**Discussion by troylynch** [music-assistant/support#1447] [Q&A] [answered]
Is there a way to use the Music Assistant app in Home Assistant to play to say my iphone remotely say in the car while driving?

**Comment by marcelveldt** [comment, marked as answer]
No that is not possible but high on the feature request list so might happen some day.

**Reply by troylynch** [reply]
Thanks! Thsi woudl be awesome. Would love to ditch itunes and have the continuity across all my devices.

**Reply by OzGav** [reply]
Also see the second item here https://github.com/orgs/music-assistant/discussions/710

**Comment by PablexXXXX** [comment]
I also think it would be awesome to be able to play in the same browser as a player or in the phone throug the android app. The media on home assistant can play using the browser as a player but cant use the radios when you are not in the same network as the home assistant server

**Reply by troylynch** [reply]
This was my hope. After looking in the above soluton with squeezebox. Another thought was maybe if music assistant could be LMS compatible some way and use squeezebox player to browse media and update playlists and play local would be nice also. I have alot of music and currently use iTunes but my libraries aren't all in iTunes and kinda all over the place. If I could organize playlists and music in Music Assistant and play remotely this would be a perfect solution. I checked out LMS which is awesome except I hate the front end I like the Music Assistant front end where it is more like a juke box front end.

**Reply by OzGav** [reply]
Not sure what you mean by LMS compatible. MA is seen as a modern replacement for LMS

**Reply by troylynch** [reply]
Just the abilkity to connect streambox players to Music Assistant to play and or manage songs. So was thinking maybe clone how LMS talks to the streambox players? Just a thought

**Reply by OzGav** [reply]
For the addition of new player providers please see if someone else has raised this in the player provider poll or post a request in there yourself and we can see how much support there is.

**Reply by troylynch** [reply]
Perfect! Thanks for the info!
